genie/process_mutation.py

```diff
@@ -226,18 +227,110 @@ def process_mutation_workflow(
         syn=syn,
         center=center,
         maf_tableid=maf_tableid,
-        annotated_maf_path=annotated_maf_path,
+        annotated_maf_path=annotation_paths.merged_maf_path,
         flatfiles_synid=flatfiles_synid,
         workdir=workdir,
     )
 
-    return annotated_maf_path
+    full_error_report = concat_annotation_error_reports(
+        center=center,
+        input_dir=annotation_paths.error_dir,
+    )
+
+    store_annotation_error_reports(
+        full_error_report=full_error_report,
+        syn=syn,
+        errors_folder_synid=genie_config["center_config"][center]["errorsSynId"],
+        release="test",
+    )
+    return annotation_paths.merged_maf_path
+
+
+def create_annotation_paths(center: str, workdir: str) -> namedtuple:
+    """Creates the filepaths required in the annotation process
+
+    Args:
+        center (str): name of the center
+        workdir (str): work directory to create paths in
+
+    Returns:
+        namedtuple: tuple with all the paths
+    """
+    input_files_dir = tempfile.mkdtemp(dir=workdir)
+    output_files_dir = tempfile.mkdtemp(dir=workdir)
+    Filepaths = namedtuple(
+        "Filepaths",
+        ["input_files_dir", "output_files_dir", "error_dir", "merged_maf_path"],
+    )
+    annotation_paths = Filepaths(
+        input_files_dir=input_files_dir,
+        output_files_dir=output_files_dir,
+        error_dir=os.path.join(output_files_dir, f"{center}_error_reports"),
+        merged_maf_path=os.path.join(
+            output_files_dir, f"data_mutations_extended_{center}.txt"
+        ),
+    )
+    return annotation_paths
+
+
+def concat_annotation_error_reports(
+    center: str,
+    input_dir: str,
+) -> pd.DataFrame:
+    """Concatenates the annotation error reports
+
+    Args:
+        center (str): name of center associated with error report
+        input_dir (str): directory where error reports are
+    Returns:
+        pd.DataFrame: full annotation error report
+    """
+    error_files = os.listdir(input_dir)
+    chunk_size = 10000
+    error_reports = []
+
+    # Read and concatenate TSV files in chunks
+    for file in error_files:
+        for chunk in pd.read_csv(
+            os.path.join(input_dir, file), sep="\t", chunksize=chunk_size
+        ):
+            error_reports.append(chunk)
+    full_error_report = pd.concat(error_reports)
```

Comment:
Nit: Querying the mutation table shows that there are ~30000 failed annotations.  Let's not prematurely optimize, but I just wanted to point out that this _could be_ an issue if there ends up being lots of annotation failures for some reason.

Reply:
Gotcha! Agreed, I was just taking precautions given that we're optimizing for when we create the compiled annotated maf file. Will remove for now.

Reply:
@rxu17 no need to remove the chunking. I'm saying there might be a day when `pd.concat` may be the issue.

Reply:
Makes sense, appending/concatenating the chunks as it goes along into a dataframe is more memory efficient than pd.concat at the end with everything at once. Then there is the side of it being possibly slower than pd.concat. But yes, something to investigate when we do get into noticeably larger annotation failures. Will put that in a confluence page for now as we have areas of the pipeline we could optimize better.